tests/st/fuzz/src/multi_kernel_test_generator.py

```
@@ -487,6 +429,16 @@ def _generate_test_class(  # noqa: PLR0912, PLR0915
             f"            TensorSpec('output', [{output_shape[0]}, {output_shape[1]}], "
             f"DataType.FP32, is_output=True),"
         )
+
+        # Add config tensor for if/else kernels
+        needs_config = orch_info.get("needs_config", False)
+        if needs_config:
+            branch_cond_val = random.randint(0, 1)
```

Ruff (0.15.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)